### PR TITLE
SW update screen scaling + default font size change

### PR DESCRIPTION
**Pull request by BenjiPestell**

### CODE REVIEW

**Comment by BenjiPestell** [comment]
Tested on 7" + 10"

File: src/asmcnc/apps/SWupdater_app/screen_update_SW.py [hunk 1]
```diff
@@ -1,722 +1,737 @@
...
+
+                        BoxLayout: 
+                            size_hint: (None, None) 
+                            orientation: "vertical"
+                            height: dp(0.208333333333*app.height)
+                            width: dp(0.06125*app.width)
+                            padding:[dp(0.00625)*app.width, dp(0.0729166666667)*app.height, dp(0.01875)*app.width, dp(0.0729166666667)*app.height]
+                            Button:
+                                font_size: str(0.01875 * app.width) + 'sp'
+                                id: refresh_button
```
Comment (LettyYeti) [inline src/asmcnc/apps/SWupdater_app/screen_update_SW.py:130]: recommend normally putting the id above font_size additions in future

File: src/asmcnc/apps/SWupdater_app/screen_update_SW.py [hunk 2]
```diff
@@ -1,722 +1,737 @@
...
-
-        self.usb_stick = usb_storage.USB_storage(self.sm, self.l)
-
-        self.sw_version_label.text = '[b]' + self.set.sw_version + '[/b]'
-        self.update_screen_with_latest_version()
-
-
-    def on_enter(self):
-
-        # Keep tabs on wifi connection
```
Comment (LettyYeti) [inline src/asmcnc/apps/SWupdater_app/screen_update_SW.py:390]: reinstate comments

File: src/asmcnc/apps/SWupdater_app/screen_update_SW.py [hunk 3]
```diff
@@ -1,722 +1,737 @@
...
-        self.check_wifi_connection(1)
-        self.poll_wifi = Clock.schedule_interval(self.check_wifi_connection, self.WIFI_CHECK_INTERVAL)
-
-        # Set up and keep tabs on usb connection
```
Comment (LettyYeti) [inline src/asmcnc/apps/SWupdater_app/screen_update_SW.py:394]: reinstate comments

File: src/asmcnc/apps/SWupdater_app/screen_update_SW.py [hunk 4]
```diff
@@ -1,722 +1,737 @@
...
+                            self.set.refresh_latest_sw_version()
+                        elif self.wifi_image.source != self.wifi_on:
+                            refresh_error_message = (
+                                self.l.get_str("Could not refresh version!")
+                                + "\n\n"
+                                + self.l.get_str(
+                                    "Please check the file on your USB stick."
+                                )
+                            )
+                            popup_info.PopupError(
```
Comment (LettyYeti) [inline src/asmcnc/apps/SWupdater_app/screen_update_SW.py:433]: ensure you make a note to revisit the popups

File: src/asmcnc/apps/SWupdater_app/screen_update_SW.py [hunk 5]
```diff
@@ -1,722 +1,737 @@
...
-                    '[b]: ' + self.set.latest_sw_version + '[/b]'
-                    )
-            else:
-                self.latest_software_version_label.text = self.l.get_bold('You are up to date!')
-        else:
-            self.latest_software_version_label.text = self.l.get_bold('No WiFi or USB!')
-
-    # Creates a popup message warning the user that the update may take a while and sets the update method as "WiFi"
-    # or "USB" depending on which button was pressed. This function is called from the Builder.load_string when the
-    # buttons are defined
```
Comment (LettyYeti) [inline src/asmcnc/apps/SWupdater_app/screen_update_SW.py:490]: reinstate comment

**Review by LettyYeti — CHANGES_REQUESTED** [review]
Mostly comments on comments

**Review by LettyYeti — APPROVED** [review]

**Pull request merged**